app/src/routes/home/index.js

```diff
@@ -4,10 +4,16 @@ const express = require("express");
 const router = express.Router();
 
 // 컨트롤러
-const ctrl = require("./home.ctrl");
+const authCtrl = require("../auth/auth.ctrl.js");
 const webtoonctrl = require("../webtoon/webtoon.ctrl");
 
-router.get("/", ctrl.output.home);
+// validation 컨트롤러
```

Comment:
큰 의미는 없는데요 정확히는 미들웨어가 맞아요

```suggestion
// validation 미들웨어
```

Reply:
피드백 감사합니다!
말씀해주신 대로 컨트롤러보다는 미들웨어가 더 정확한 표현인 것 같네요!